rllib/core/rl_module/rl_module.py

```diff
@@ -307,6 +320,12 @@ def __post_init__(self):
             self.output_specs_inference()
         )
 
+    def setup(self):
+        """Sets up the components of the module.
+
+        This is called automatically during the __init__ method of the subclass. This abstraction can be used to create any component that you RLModule needs.
```

Comment:
To be clear:
Since this is pretty important for users, I think we should be clear here:
It's called from the __init__ method of this class - therefore, the subclass _should_ call super init.